source/MaterialXGenMdl/mdl/materialx/stdlib_1_6.mdl

```diff
@@ -2872,6 +2872,66 @@ export core::color4 mx_screen_color4(
 	return core::color4(rgb,a);
 }
 
+float mx_overlay(float mxp_fg, float mxp_bg)
```

Comment:
Similarly, I think this MDL function for the overlay node may be from an _earlier_ version of MaterialX, and it was removed when Rasmus Bonnedal implemented the overlay node as a language-independent graph:

https://github.com/AcademySoftwareFoundation/MaterialX/pull/1539

Reply:
Thanks. Removed mx_overlay functions.